### PR TITLE
Daily code

### PR DESCRIPTION
**Pull request by celineboudaie**
Test

### CODE REVIEW

**Review by xuanyangge — COMMENTED** [review]
test comment

File: capstone/capstone/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -136,8 +126,18 @@
...
+                                                    <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+                                                    <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                                    <nil key="textColor"/>
+                                                    <nil key="highlightedColor"/>
+                                                </label>
                                             </subviews>
                                         </tableViewCellContentView>
+                                        <connections>
+                                            <outlet property="productName" destination="wlo-8w-cTR" id="JTb-6d-VRE"/>
+                                        </connections>
```
Comment (xuuuge) [inline capstone/capstone/Base.lproj/Main.storyboard:133]: test comment

**Review by xuanyangge — APPROVED** [review]

**Review by xuanyangge — APPROVED** [review]

**Comment by celineboudaie** [comment]
> Curious, how did you find this id "P297132"?

I accidentally made a commit with the wrong link, but that is the product id since I was attempting to print out the product name in the cell

**Comment by celineboudaie** [comment]
Implementing API in cell

**Pull request merged**